features/features/package_managers/rebar_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'feature_helper'
+
+describe "Rebar Dependencies" do
+  # As a Node developer
+  # I want to be able to manage NPM dependencies
+
+  let(:erlang_developer) { LicenseFinder::TestingDSL::User.new }
+
+  specify "are shown in reports" do
+    LicenseFinder::TestingDSL::RebarProject.create
+    erlang_developer.run_license_finder
+    expect(erlang_developer).to be_seeing_line "sqerl, \"BRANCH: master\", \"Apache 2.0\""
```

Comment:
This is the only mention of `sqerl` ... is `sqerl` a dependency of all `rebar`-managed projects?  Or is it missing from `spec/fixtures/all_pms/rebar.config`?